README.md

```diff
@@ -51,12 +136,24 @@ pip install pytest
 pytest test
 ```
 
-TODO
------
+Migration Guide
+----------------
 
-Get the docs on read the docs
+The `setgapth.py` way to install is now deprecated by `pip install galgebra` and all modules in GAlgebra should be imported from `galgebra`, for example:
 
-Ensure functions are documented
+```python
+from galgebra.printer import Format, Eprint, Get_Program, latex, GaPrinter
+from galgebra.ga import Ga, one, zero
+from galgebra.mv import Mv, Nga
+# for backward compatibility
+from galgebra.mv import MV, ONE, ZERO, HALF
+from galgebra import ga
+from galgebra import utils
+```
 
-Set up proper release version system
+Resources
+------------
 
+Note that in the doc directory there is BookGA.pdf which is a collection of notes on 
```

Comment:
I don't think the "Note" is necessary here. Also, you probably want to mention the `doc/books` instead of just `doc`.

We can even link the path as `[doc/books/](doc/books/)`. Such a relative link is valid markdown and works on GitHub, but if you prefer the absolute link, it chould be something like:

`[doc/books/](https://github.com/pygae/galgebra/tree/master/doc/books)`

(Not sure how/if this should be adapted to display correctly in readthedocs.)

Reply:
This is in brombo's original README and temporarily placed here, will remove it and replace it with a detailed Resources section soon.

Reply:
Left for later PRs.